stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -116,12 +114,23 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 PaymentOptionResult.Failed(it)
             )
         }
+
+        // Enable animation for layout transitions
+        bottomSheet.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
+        fragmentContainerParent.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
+
         bottomSheetController.shouldFinish.observe(this) { shouldFinish ->
             if (shouldFinish) {
                 finish()
             }
         }
+
         bottomSheetController.setup()
+        // Fragment animates in immediately after activity is created, so we can already expand it
```

Comment:
`Fragment animates immediately after activity is created, so we can already expand it`

Reply:
I meant that it's added with animation. Changed wording.